### ISSUE TITLE
`no-setup-in-describe` triggers on `describe.skip`

### ISSUE THREAD

**Issue by edg2s** [lo1tuma/eslint-plugin-mocha#249]
```js
describe.skip( 'My test', () => {
  it( 'Something', () => {
    // Foo
  } );
} );
```

This should not trigger `no-setup-in-describe`, yet it does.

This can be worked around by moving the `skip` to each of the `it` calls.

**Comment by lo1tuma** [comment]
Can you explain why it shouldn’t trigger? If the test code is not ready or somehow broken it should be removed. If it is important enough to keep the test code then it should be also important enough to refactor and move the setup code to the appropriate places.

**Comment by edg2s** [comment]
The rule should only trigger when there is setup code in the describe, and there isn't any.

**Comment by lo1tuma** [comment]
There is a `describe` block in your example.

**Comment by lo1tuma** [comment]
I guess I understand the issue now. I thought this issue was a feature request to skip checking for setup code when using `describe.skip()`, but it is actually a bug, where `describe.skip()` triggers this rule even when there is no setup code.

Thanks for reporting. That is definitely a bug.
